### ISSUE TITLE
Can I turn this Repo Private? @TarekElalfi

### ISSUE THREAD

**Issue by alexandrepayumo** [alexandrepayumo/soen342_project_payumo_khattak#3]
Hi @TarekElalfi , can I turn this repo private, I would like to hide the admin credentials in this README from the public

**Comment by TarekElalfi** [comment]
Yes, you can turn it private unless said otherwise on the project description or by the professor.
